src/Writer.php

```diff
@@ -79,6 +82,8 @@ public function open($export)
     {
         $this->exportable = $export;
 
+        $this->clearListeners();
```

Comment:
Would clearing be needed if Writer wouldn't be a singleton?

Reply:
No, it would not.